front/cypress/e2e/email_consent.cy.ts

```diff
@@ -42,16 +41,22 @@ describe('email consent', () => {
       .contains('Unsubscribe')
       .click();
 
-    cy.wait(2000);
-
-    cy.get('#e2e-custom-email-container iframe').then(($iframe) => {
-      const $body = $iframe.contents().find('body');
-      cy.wrap($body).find('#e2e-email-settings-page');
-      cy.wrap($body).find('#e2e-consent-form');
-      cy.wrap($body)
-        .find('.e2e-unsubscribe-status')
-        .contains('success')
-        .contains('Official');
-    });
+    // TODO: After re-enabling this test (it was being skipped) a few weeks back, I've tried many different ways to get these
+    // checks to pass, but I'm unable to do so. Working with iframes is known to be pretty tricky in cypress.
+    // I'm commenting this out for now, but we should try to get this working in the future.
+    //   cy.get('#e2e-custom-email-container iframe').should('exist');
```

Comment:
This is the only part that fails on CI now (it always passes locally). It was a skipped test that I re-enabled a few weeks ago, but I've tried everything I can think of to get this passing, but it won't. Locally it passes fine, but there is an issue with handling the iframes on CI which I can't figure out. Iframes in Cypress are also known to be buggy/complicated.

I would rather move on to other work for now, so I've just left a TODO here.

Reply:
Yes, agree